SERVER/src/character_server.py

```
@@ -1,43 +1,181 @@
+    # This test requires a DB with Actor1 configured, and models.
+    # It's more of an integration test component.
+    async def test_character_server():
+        print("Testing CharacterServer (Actor1)...")
+        # Mock DB or ensure DB_PATH points to a test DB with Actor1
+        class DummyDB:
+            def get_character(self, Actor_id):
+                if Actor_id == "Actor1":
+                    return {"name": "TestActor1", "personality": "tester", "tts": "gtts", "language":"en",
+                            "reference_audio_filename": None, "Actor_id": "Actor1", "llm_model": None} # Use gTTS for no model download
+                return None
+            def save_training_data(self, data, Actor_id): print(f"DummyDB: Save training data for {Actor_id}: {data}")
```

Ruff (0.11.9)

Missing return type annotation for private function `save_training_data`

Add return type annotation: `None`

(ANN202)